src/applications/disability-benefits/view-payments/components/view-payments-lists/payments/Payments.jsx

```diff
@@ -76,13 +132,14 @@ class PaymentsReceived extends Component {
           </p>
           <ResponsiveTable
             className="va-table"
-            currentSort={{
-              value: 'String',
-              order: 'ASC',
-            }}
             fields={this.props.fields}
             data={this.state.currentlyShowingData}
             maxRows={10}
+            onHeaderClick={(value, order) => this.handleSort(value, order)}
+            currentSort={{
+              value: this.state.sortingOn,
+              order: this.state.currentSortOrder,
+            }}
```

Comment:
I haven't used this component before... it doesn't look like it updates the `aria-sort` attribute. This might need to also be updated.

Reply:
@Mottie @jenstrickland Added code for this [here](https://github.com/department-of-veterans-affairs/vets-website/pull/13389/commits/058170b766dc1027d0718af43f97d9f739b2f553)